hermes.toml

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR)
+#
+# SPDX-License-Identifier: CC0-1.0
+
+[harvest]
+from = [ "cff", "git" ]
+
+[harvest.cff]
+validate = false
+
+[process]
+preprocess = [ "cff", "git" ]
+
+[deposit]
+mapping = "invenio"
+target = "invenio"
+
+[deposit.invenio]
+base_url = "https://sandbox.zenodo.org"
+auth_token = "UGOJ9fwRXUhshyWzQtqAqHKQRHlT6VPUrUNRV9v984zkjUvjhwFo8SnVYtXx"
```

Comment:
This is of cause not a valid and working auth token (anymore)

Reply:
I don't think it would be a good idea to allow users to configure the token in the config file. The config file will be commited to the repo and this will result in accidentally commited auth tokens. Only using command line options and environment variables should be the way to go.

Reply:
I totally agree. 😊

Reply:
I completely and strongly agree. These secrets should ideally be read from an env var or some file. All CI systems expose secrets as env vars, so probably that would be fine. It might be nice to configure the name of the env var to read, though.